backend/prisma/README.md

```diff
@@ -1,6 +1,11 @@
-https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgres
+Prefix all commands to access the correct .env file with: dotenv -e ../.env
+
+Connecting DB: https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgres
+- 1. Replace the database connection string in the schema.prisma with your own.
```

Comment:
this should actually be replaced in side the `.env` file instead!

Reply:
okay i will rrevise that too